scd30/scd30.c

```diff
@@ -82,19 +82,19 @@ int16_t scd30_read_measurement(float* co2_ppm, float* temperature,
 
     ret =
         sensirion_i2c_write_cmd(SCD30_I2C_ADDRESS, SCD30_CMD_READ_MEASUREMENT);
-    if (ret != STATUS_OK)
+    if (ret != NO_ERROR)
```

Comment:
do you also want to rename `ret` to `err` then?

Reply:
Renamed to `error`.